Adding Page for ml5 Object Detection Video (#2745)
CodingTrain/website: _learning/ml5/9.1-object-detection.md

---
title: "Object Detection with COCO-SSD"
video_number: 9.1
date: 2020-10-14
video_id: QEzRxnuaZCk
repository: ml5/9.1-object-detection
can_contribute: true

variations:
  - name: "object detection (image)"
    folder: "1_image"
    lang: "p5js"
    web_editor: ZNQQx2n5o
  - name: "object detection webcam"
    folder: "2_webcam"
    lang: "p5js"
    web_editor: VIYRpcME3
  - name: "object detection webcam persistence"
    folder: "3_webcam_persistence"
    lang: "p5js"
    web_editor: Vt9xeTxWJ

links:

>   - title: "Object Detector ml5.js Reference"

Reviewer: ```suggestion
  - title: "Object Detector ml5.js Reference" (coming soon)
```

Author: Added link to the markdown ml5 page.

    url: 

>   - title: "COCO ml5.js Model and Data Provenance"

Reviewer: ```suggestion
  - title: "COCO ml5.js Model and Data Provenance" (coming soon)
```

Author: Added link to the markdown ml5 page.

  - title: "COCO 2020 Keypoint Detection Task"
    url: https://cocodataset.org/index.htm#keypoints-2020
  - title: "The COCO-Stuff dataset"
    url: https://github.com/nightrome/cocostuff
  - title: "Humans of AI (Editorial Essay)"
    author: "Philipp Schmitt"
    url: https://humans-of.ai/editorial/
  - title: "Excavating AI - The Politics of Images in Machine Learning Training Sets"
    author: "Kate Crawford and Trevor Paglen"
    url: https://www.excavating.ai/
  - title: "Object Detection (coco-ssd) Model on tensorflow.js Models GitHub Repository"
    url: https://github.com/tensorflow/tfjs-models/tree/master/coco-ssd
  - title: "List of Classes Available in COCO Dataset (ml5.js)"
    url: https://github.com/ml5js/ml5-library/blob/development/src/utils/COCO_CLASSES.js
  - title: "Eevee Mango & Greta Goose Instagram"
    url: https://www.instagram.com/mangoandgoose/

videos:
  - title: "ml5.js Pose Estimation with PoseNet"
    author: "The Coding Train"
    url: "/learning/ml5/7.1-posenet"
  - title: "Coding in the Cabana 4 - Worley Noise"
    author: "The Coding Train"
    url: "/challenges/coding-in-the-cabana/004-worley-noise"
---

> In this video I check out the new object detection in ml5.js that uses the tensorflow.js coco-ssd model. Follow along to make a project that can detect 80 types of objects in an image or video!

Reviewer: ```suggestion
In this video, I cover object detection in ml5.js with the COCO-SSD pre-trained model.
```
